sanic/config.py

```diff
@@ -6,10 +6,28 @@
 
 SANIC_PREFIX = "SANIC_"
 
+DEFAULT_CONFIG = {
+    "REQUEST_MAX_SIZE": 100000000,  # 100 megabytes
+    "REQUEST_BUFFER_QUEUE_SIZE": 100,
+    "REQUEST_TIMEOUT": 60,  # 60 seconds
+    "RESPONSE_TIMEOUT": 60,  # 60 seconds
+    "KEEP_ALIVE": True,
+    "KEEP_ALIVE_TIMEOUT": 5,  # 5 seconds
+    "WEBSOCKET_MAX_SIZE": 2 ** 20,  # 1 megabytes
+    "WEBSOCKET_MAX_QUEUE": 32,
+    "WEBSOCKET_READ_LIMIT": 2 ** 16,
+    "WEBSOCKET_WRITE_LIMIT": 2 ** 16,
+    "GRACEFUL_SHUTDOWN_TIMEOUT": 15.0,  # 15 sec
+    "ACCESS_LOG": True,
+}
+
 
 class Config(dict):
-    def __init__(self, defaults=None, load_env=True, keep_alive=True):
+    def __init__(self, defaults=None, load_env=True, keep_alive=None):
```

Comment:
Default value for keep_alive was moved in DEFAULT_CONFIG. If keep_alive is passed explicitly - we will use it. Otherwise even if you pass KEEP_ALIVE in defaults and don't pass it as kwarg it will be rewrited by default value. e.g.
`
config = Config(defaults={"KEEP_ALIVE": False})
config.KEEP_ALIVE == True

config = Config(defaults={"KEEP_ALIVE": False}, keep_alive=False)
config.KEEP_ALIVE == False
`

Reply:
I see it now. That's what I get for doing a code review on my phone.